fec.config.js

```diff
@@ -0,0 +1,28 @@
+const { resolve } = require('path');
+
+module.exports = {
+  appUrl: '/insights/tasks',
+  debug: true,
+  useProxy: process.env.PROXY === 'true',
+  proxyVerbose: true,
```

Comment:
Let's stick to the fec.config.js implementation in the rest of the apps and remove this field as well.

Reply:
I find this useful as it shows me what is happening with the requests in the terminal.